### PR TITLE
Update for bleach 2, add travis.ci

### PR DESCRIPTION
**Pull request by dgilman**
Get the module off of the dead-end bleach 1.x series and avoid a bit of pain around html5lib.  There is also a travis.ci config that runs the unit tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+   - "2.7"
+install: "pip install -r requirements.txt"
+script: python ./wikimarkup/tests.py
+after_success: python setup.py bdist_egg
```
Comment (erikrose) [inline .travis.yml]: Is this just to test whether the egg build succeeds? In any case, eggs are fairly deprecated. I'd recommend removing this line.

**Review by erikrose — CHANGES_REQUESTED** [review]
Otherwise, this looks good to me! Thanks for the patch! :-D

**Comment by dgilman** [comment]
Yes, I wanted to give the built artifact a test.  What about the package version - do you think that should be bumped?

**Comment by erikrose** [comment]
Gotcha. Building the egg would have proven that the egg built without crashing, but it wouldn't have tested that it actually did the right thing. It's not insane to run the tests against a build sdist (for instance) to make sure the MANIFEST.in is including what it needs to, but I've rarely seen it done. The Travis config looks good as it to me now.

It's up to you whether to bump the version. We don't own the [PyPI entry](https://pypi.python.org/pypi/py-wikimarkup/), and this form has never been published to PyPI, so I don't know that there's much point.

**Pull request merged**